src/files.cpp

```diff
@@ -17,9 +18,12 @@
 #include <string>
 
 #include "main.hpp"
+#include "files.hpp"
 #include "sound.hpp"
 #include "entity.hpp"
 
```

Comment:
Windows needs `direct.h` for `mkdir()`

Reply:
Also, Microsoft has some warning about mkdir: https://docs.microsoft.com/en-us/cpp/c-runtime-library/reference/mkdir